profile/rwahs.xml

```diff
@@ -7779,10 +7779,13 @@
         <permission group="memorial" access="read"/>
       </groupAccess>
       <bundlePlacements>
-        <placement code="preferred_labels">
-          <bundle>ca_object_labels.name</bundle>
+        <placement code="ca_objects_preferred_labels">
+          <bundle>ca_objects.preferred_labels</bundle>
           <settings>
             <setting name="label" locale="en_AU">Title</setting>
+            <setting name="format"><![CDATA[<nobr>^ca_objects.preferred_labels.name</nobr>]]></setting>
```

Comment:
@kehh This might not be the best way, is there something like `^value` to get the bundle value?  Or would it be `^name`?  Thought I'd just do something that works and then ask you what's the best practice.